django/website/contacts/models.py

```diff
@@ -129,6 +129,8 @@ class User(AbstractBaseUser, PermissionsMixin):
         blank=True,
         null=True)
 
+    last_viewed_logframe = ForeignKey('logframe.LogFrame', null=True)
```

Comment:
I'm not sure this should be stored in the User model. 

If I was to export users, I wouldn't want to see this value. It feels like temporary information we wouldn't even need to store permanently - other than to provide a default when the user logs in.

I can't decide whether this should be a "fine for now, will refactor when we have a better idea of what user sessions/preferences we want" or whether we should rethink it now.